tests/quantizer_impl_test.py

```diff
@@ -283,15 +283,15 @@ def test_GetScale_ElementsPerScale_Scalar_ScaleAxis_EPS():
   # values and the input x and q tensors have rank 4
   x_r4 = tf.random.uniform([2, 4, 8, 16])
   q_r4 = tf.random.uniform([2, 4, 8, 16])
-  scale_r4_eps_none_ua_none = quantizers._get_scale(
+  scale_r4_eps_none_ua_none = quantizers._get_least_squares_scale(
       "auto", x_r4, q_r4, elements_per_scale=None, scale_axis=None)
-  scale_r4_eps_2_ua_0 = quantizers._get_scale(
+  scale_r4_eps_2_ua_0 = quantizers._get_least_squares_scale(
       "auto", x_r4, q_r4, elements_per_scale=2, scale_axis=0)
-  scale_r4_eps_2_ua_1 = quantizers._get_scale(
+  scale_r4_eps_2_ua_1 = quantizers._get_least_squares_scale(
       "auto", x_r4, q_r4, elements_per_scale=2, scale_axis=1)
-  scale_r4_eps_2_ua_2 = quantizers._get_scale(
+  scale_r4_eps_2_ua_2 = quantizers._get_least_squares_scale(
       "auto", x_r4, q_r4, elements_per_scale=2, scale_axis=2)
-  scale_r4_eps_2_ua_3 = quantizers._get_scale(
+  scale_r4_eps_2_ua_3 = quantizers._get_least_squares_scale(
       "auto", x_r4, q_r4, elements_per_scale=2, scale_axis=3)
 
   assert_equal(tf.shape(scale_r4_eps_none_ua_none).numpy(), [1, 1, 1, 16])
```

Comment:
consider add assertion on some specific scale values as well. The same goes for other related tests.

Reply:
See discussion above